media/css/firefox/all.scss

```diff
@@ -124,7 +124,7 @@ $image-path: '/media/protocol/img';
             ));
         background-size: 1.5em auto;
         background-repeat: no-repeat;
-        margin-bottom: 0.6rem;
+        padding-bottom: 0.6rem;
```

Comment:
The actual fix is also just a one-liner ^^ ~ the other added spacing based on device size was just thrown in when testing, seeing some ugly line wraps while there was space available on the other side.